src/lib/tabs/tab-nav-bar/tab-nav-bar.html

```diff
@@ -0,0 +1,4 @@
+<div class="md-tab-header">
```

Comment:
I'm thinking that this should probably be a `<nav>` element instead of a div. 

@marcysutton Is it appropriate to always use a `<nav>` element for a component that contains anchors and navigates the page, or should `<nav>` only be used for _major_ site navigation?